content/200-concepts/100-components/02-prisma-client/260-client-extensions/index.mdx

```diff
@@ -77,28 +77,46 @@ Example uses of isolated client instances include the following:
 
 ### Declare multiple extensions
```

Comment:
Is this really "declaring" extentions? Is it not applying them to a client?
The "declaring" has happened where `extensionA` and `extensionB` have been defined I feel.

Reply:
Point taken!